ui/client/components/graph/node-modal/MapVariable.js

```diff
@@ -1,65 +1,57 @@
-import PropTypes from "prop-types";
-import Input from "./Input";
-import Textarea from "./Textarea";
-import {v4 as uuid4} from "uuid";
-import React from "react";
-import _ from "lodash";
-import Map from "./editors/map/Map";
-import {errorValidator, notEmptyValidator} from "../../../common/Validators";
+import PropTypes from "prop-types"
+import LabeledTextarea from "./editors/field/LabeledTextarea"
+import {v4 as uuid4} from "uuid"
+import React from "react"
+import _ from "lodash"
+import Map from "./editors/map/Map"
+import {errorValidator, notEmptyValidator} from "../../../common/Validators"
+import LabeledInput from "./editors/field/LabeledInput"
 
 const MapVariable = (props) => {
```

Comment:
I think in future we should give another name for these components with postfix Node, eq. Map VariableNode. Now we have Map, Field, etc .. It's difficult to understand that.

Reply:
Yes, we should think about that